_variables.tf

```diff
@@ -202,7 +202,7 @@ variable "family" {
 variable "create_db_option_group" {
   description = "(Optional) Create a database option group"
   type        = bool
-  default     = true
+  default     = false
```

Comment:
Would it be a breaking change as It may remove option groups previously created? just curious.

Reply:
No, because this option was never used despite being declared in the variables.